diplo.lua

```diff
@@ -0,0 +1,128 @@
+@@ -0,0 +1,79 @@

```

Comment:
this looks like merge error from diff text

Reply:
Yep. Pushing things by hand can get messy.